### ISSUE TITLE
Change sidebar from aside to nav?

### ISSUE THREAD

**Issue by rshipp** [BanchanArt/banchan#551]
**Issue**: Navigation should be inside a `nav` element
**Impact**: Low
**Recommendation**: Change this `aside` to a `nav`. Screenreader users may use “landmarks” to navigate a webpage, and expect to find these links in a navigation landmark. There seems to be some precedent for using `aside` for sidebars, so this might be more of a best-practice than a big issue.
**More info**: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav, https://dequeuniversity.com/checklists/web/landmarks 
**Screenshot**:
![image](https://github.com/BanchanArt/banchan/assets/1382374/c38009ff-f695-40c3-9c1d-d27289d5aa06)
